### PR TITLE
Add logo to US Election 2016 stream

### PR DESCRIPTION
**Pull request by sumeetadur**

### CODE REVIEW

**Comment by sumeetadur** [comment]
@callumlocke

File: client/components/sub-header/_main.scss [hunk 1]
```diff
@@ -94,6 +95,43 @@
...
+			font-size: $font-size;
+			margin: 0 $horizontal-spacing 0 ($logo-size + $horizontal-spacing);
+
+			&:before {
+				$image-size: ($logo-size * 2) / 1px;
+
+				height: $logo-size;
+				width: $logo-size;
+				left: -($logo-size + $horizontal-spacing);
+				background-image: url('https://ig.ft.com/_other/us-election-2016-stream/logo.svg');
```
Comment (callumlocke) [inline client/components/sub-header/_main.scss]: for the record: we're hoping to find a suitable place for this kind of graphic in next/origami land, and going to discuss it on thursday (see https://github.com/Financial-Times/o-ft-icons/issues/123)

but until then we can host it on our IG server.

**Comment by callumlocke** [comment]
This looks good to me. @i-like-robots is this OK?

**Comment by i-like-robots** [comment]
Just add a screenshot to the PR for reference.

If I was going to be _really_ fussy I'd probably put these styles in a separate partial but I don't mind really.

**Pull request merged**